airflow/Dockerfile

```diff
@@ -16,3 +16,8 @@ RUN pip3 install \
   apache-airflow[statsd]==${AIRFLOW_VERSION} \
   influxdb==${INFLUX_DB_VERSION} \
   werkzeug==${WERKZEUG_VERSION}
+
+RUN pip install \
+ --upgrade\
+ boto3 \
```

Comment:
Can you add a specific version? This will make this stop working if there's a major release that breaks API compatibility

Reply:
Done, using the same versions as we do in Jupyter.